DIGIT210/classwork/akam-python-test.py

```diff
@@ -13,6 +13,9 @@
 
 # nlp = spacy.cli.download("en_core_web_lg")
 nlp = spacy.load('en_core_web_lg')
+
+souls = 'dsexcel-text-test.xml'
+
 def readTextFiles(filepath):
```

Comment:
@tylerakam This is the first `readTextFiles()` function. It was the *only* one that was working before. We don't really need it now and I don't think it's actually firing. (You can probably delete it.) The *second* `readTextFiles()` function further on in the file is the one doing the work of pulling from the XML document with XPath and delivering those strings